script.js

```diff
@@ -1,4 +1,180 @@
+    myOutputValue = "Welcome Player 1. <br> You Rolled";
+    var displayCounter1 = 0;
+    while (displayCounter1 < numberofDice - 1) {
+      myOutputValue =
+        myOutputValue +
+        " " +
+        player1Array[displayCounter1] +
+        " for Dice " +
+        (displayCounter1 + 1) +
+        ", ";
+      displayCounter1 += 1;
+    }
+    myOutputValue =
+      myOutputValue +
+      " and " +
+      player1Array[player1ArrayLength - 1] +
+      " for Dice " +
+      player1ArrayLength +
+      ". <br> Choose the order of the dice.";
+    return myOutputValue;
+  }
+  //when player 1 chooses which dice, create the player's combined number
+  if (input == 1 && player1RoundCounter == player2RoundCounter) {
+    player1CombinedNumber = concatenate2Numbers(
+      player1Array[0],
+      player1Array[1]
+    );
+    player1RoundCounter += 1;
+    console.log("Player 1 round counter: " + player1RoundCounter);
+    return (
+      "Player 1's number is " +
+      player1CombinedNumber +
+      "<br> It is now Player 2's turn."
+    );
+  }
+
+  if (input == 2 && player1RoundCounter == player2RoundCounter) {
+    player1CombinedNumber = concatenate2Numbers(
+      player1Array[1],
+      player1Array[0]
+    );
+    player1RoundCounter += 1;
+    console.log("Player 1 round counter: " + player1RoundCounter);
+    return (
+      "Player 1's number is " +
+      player1CombinedNumber +
+      "<br> It is now Player 2's turn."
+    );
+  }
+
+  //if player 1 has gone, then now its player 2's turn to roll the dice
+  if ((input == "") & (player1RoundCounter > player2RoundCounter)) {
+    player2Array = createDiceRollArray(numberofDice);
+    var player2ArrayLength = player2Array.length;
+    myOutputValue = "Welcome Player 2. <br> You Rolled";
+    var displayCounter2 = 0;
+    while (displayCounter2 < numberofDice - 1) {
+      myOutputValue =
+        myOutputValue +
+        " " +
+        player2Array[displayCounter2] +
+        " for Dice " +
+        (displayCounter2 + 1) +
+        ", ";
+      displayCounter2 += 1;
+    }
+    myOutputValue =
+      myOutputValue +
+      " and " +
+      player2Array[player2ArrayLength - 1] +
+      " for Dice " +
+      player2ArrayLength +
+      ". <br> Choose the order of the dice.";
+    return myOutputValue;
+  }
+
+  //when player 2 chooses which dice, create the player's combined number
+  //determine who won the game
+  if (input == 1 && player1RoundCounter > player2RoundCounter) {
+    player2CombinedNumber = concatenate2Numbers(
+      player2Array[0],
+      player2Array[1]
+    );
+    player2RoundCounter += 1;
+    console.log("Player 2 round counter: " + player2RoundCounter);
+    myOutputValue =
+      "Player 2's number is " +
+      player2CombinedNumber +
+      "<br> Player 1's Number was " +
+      player1CombinedNumber;
+    if (player1CombinedNumber > player2CombinedNumber) {
+      myOutputValue = myOutputValue + "<br> Player 1 won!";
+    } else {
```

Comment:
Good use of `else` to 1. Control the code flow, 2. Highlight to the reader that there's a mutually exclusive relationship between the code blocks, and 3. Serve as a catch-all